.env.development

```diff
@@ -3,6 +3,7 @@ ZETKIN_GEN2_ORGANIZE_URL=http://organize.dev.zetkin.org
 ZETKIN_GEN2_CALL_URL=http://call.dev.zetkin.org
 
 ZETKIN_PRIVACY_POLICY_LINK=https://zetkin.org/privacy
+ORGANIZATION_PRIVACY_POLICY_LINK=https://zetkin.org/privacy
```

Comment:
I don't think we need a separate link for this. I believe that this could have just been the `ZETKIN_PRIVACY_POLICY_LINK`. Both are effectively the _instance owner_ privacy policy.